ConsoleApp/School/Person.cs

```diff
@@ -0,0 +1,16 @@
+namespace ConsoleApp.School;
+
+class Person
+{
+    public string FirstName { get; init; }
+    public string LastName { get; init; }
+    private string email { get; set; }
```

Comment:
code convention violated here
```suggestion
    private string Email { get; set; }
```